### ISSUE TITLE
Firebase logins included in README

### ISSUE THREAD

**Issue by Chrimle** [DavidZamanian/Bachelor-thesis-blockchain-for-medical-records#74]
In the final report, we reference this repo and "how to install and use it". However, the README currently lacks critical information: the firebase login credentials. This means we need to disclose this info somehow.

**Comment by DavidZamanian** [comment]
You mean login to the application right? Not the gmail account to firebase?

**Comment by Chrimle** [comment]
Yes, exactly. For patients and doctors.

Also... with that... anyone could change a patient's contact info :joy:

**Comment by DavidZamanian** [comment]
Yeah that would be a problem, especially when our project is public 😅 Maybe we could create a test user that is not allowed to change info, or maybe allowed to change but reset after the session is done?

**Comment by Chrimle** [comment]
Perhaps we stall this until the report is published/submitted?
So, during development, but also while demo:ing, recording for the video or during a presentation etc.
After that we could it go berserk :wink: 

Making significant changes like that seems unwarranted.

**Comment by DavidZamanian** [comment]
Yes that is probably for the best XD
